src/PlusLibConfig.cmake.in

```diff
@@ -61,6 +61,15 @@ IF(PLUS_USE_INTERSONSDKCXX_VIDEO)
   FIND_PACKAGE(IntersonSDKCxx REQUIRED)
 ENDIF()
 
+# Get the location for IntersonArrayCxx
+SET(PLUS_USE_INTERSONARRAYSDKCXX_VIDEO "@PLUS_USE_INTERSONARRAYSDKCXX_VIDEO@")
```

Comment:
Ah, I see the other IntersonSDKCxx does this as well. If you're feeling generous, could you remove the other one as well?